.github/workflows/ci.yml

```diff
@@ -78,10 +78,26 @@ jobs:
           docker --version;
       - name: Test basic example
         working-directory: test/examples
-        run: docker-compose -f basic.docker-compose.yml up --build --exit-code-from test
+        run: docker-compose -f e2e.docker-compose.yml -f published.override.yml up --exit-code-from test
       - name: Test express example
         working-directory: test/examples
-        run: docker-compose -f express.docker-compose.yml up --build --exit-code-from test
+        run: docker-compose -f e2e.docker-compose.yml -f express.override.yml -f published.override.yml up --exit-code-from test
       - name: Test mixed example
         working-directory: test/examples
-        run: docker-compose -f mixed.docker-compose.yml up --build --exit-code-from test
+        run: docker-compose -f e2e.docker-compose.yml -f mixed.override.yml -f published.override.yml up --exit-code-from test
+
+  e2e:
+    runs-on: ubuntu-latest
+    permissions: read-all
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Test basic example
+        working-directory: test/examples
+        run: docker-compose -f e2e.docker-compose.yml up --exit-code-from test
```

Comment:
Can we run just the Collector in a separate container, and have test suite run from the base VM that GH Actions launch? For example see <https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/.github/workflows/unit-test.yml>